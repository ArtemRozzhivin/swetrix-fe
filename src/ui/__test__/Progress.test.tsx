/* eslint-disable testing-library/prefer-screen-queries */
/* eslint-disable import/no-extraneous-dependencies */
import React from 'react'
import { render } from '@testing-library/react'

import Progress from '../Progress'

describe('Progress', () => {
  it('should render', () => {
    const { container } = render(<Progress now={50} />)
    expect(container).toMatchSnapshot()
  })
})
