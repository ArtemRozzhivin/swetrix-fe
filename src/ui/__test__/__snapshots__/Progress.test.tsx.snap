// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Progress should render 1`] = `
<div>
  <div
    class="relative"
  >
    <div
      class="overflow-hidden h-2 text-xs flex rounded bg-blue-200 dark:bg-gray-500"
    >
      <div
        class="shadow-none flex flex-col text-center whitespace-nowrap text-white justify-center bg-blue-500 dark:bg-blue-700"
        style="width: 50%;"
      />
    </div>
  </div>
</div>
`;
